main.go

```diff
@@ -119,6 +125,19 @@ func (configs ConfigsModel) validate() error {
 	if err := input.ValidateIfPathExists(configs.ZipPath); err != nil {
 		return fmt.Errorf("Issue with ZipPath: %s", err)
 	}
+	if err := validateTimeout(configs.TestTimeout); err != nil {
+		return fmt.Errorf("Issue with TestTimeout: %s", err)
+	}
+
+	return nil
+}
+
+func validateTimeout(duration string) error {
```

Comment:
I think naming is confusing, I would expect a method that checks if the given timeout is valid or not, but this only check if the given input has the correct format, so I recommend to rename this to something more specific, like validateTimeoutFormat